api/config/service/auditsink/slack/v1/slack.proto

```diff
@@ -13,4 +13,17 @@ message SlackConfig {
   string channel = 2 [ (validate.rules).string = {min_bytes : 1} ];
 
   clutch.config.service.audit.v1.Filter filter = 3;
+  Override override = 4;
+}
+
+message CustomSlackMessage {
+  // The full method in the format of `/SERVICE/METHOD`
+  string full_method = 1 [ (validate.rules).string = {min_bytes : 1} ];
+  // The custom slack message for `/SERVICE/METHOD`
```

Comment:
i would give slightly more detail in the doc here about how it's a template. obviously can write much more thorough docs once we settle on the approach for templating so maybe a TODO works as well.

also a link to slack's formatting docs would be nice.

Reply:
added more details and left a to-do to expand on it (& docs to clutch.sh) after the transformer PR goes in